ibm/mas_devops/playbooks/mas/configure-suite-db.yml

```diff
@@ -9,6 +9,7 @@
     jdbc_instance_password: "{{ lookup('env', 'MAS_JDBC_PASSWORD') }}"
     jdbc_url: "{{ lookup('env', 'MAS_JDBC_URL') }}"
     db_pem_file: "{{ lookup('env', 'MAS_JDBC_CERT_LOCAL_FILE') }}"
-    jdbc_template: "{{ lookup('env', 'MAS_JDBC_CONFIG') | default('templates/jdbccfg-workspace-application.yml.j2', true) }}"
+    mas_config_scope: "{{ lookup('env', 'MAS_CONFIG_SCOPE) | default('wsapp', True)}}"
```

Comment:
I don't think this (mas_config_scope) should have a default.  User should always specify the scope as there's no sensible default value for this, every situation will be different what the best default would be